readthedocs/templates/support.html

```diff
@@ -0,0 +1,75 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load core_tags %}
+
+
+{% block title %}Support{% endblock %}
+
+{% block content %}
+<div id="support_page">
+{% filter restructuredtext %}
+
+Usage Questions
+~~~~~~~~~~~~~~~~
+
+If you have questions about how to use Read the Docs well,
```

Comment:
Describing what we consider as "usage" would be good here, similar to our prospectus. I don't want to confuse users into submitting actual issues to stack overflow.